src/Umbraco.Core/Persistence/Repositories/DataTypeDefinitionRepository.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
:wink: this is pretty good explanation:
https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

The basic idea is that you do an interactive rebase and pick number of previous commits `git rebase -i HEAD~3` then pick 'squash' (for each commit) in the text editor. 
When pushing to the remote you might have to use force `git push --force origin master`